apps/prairielearn/src/pages/partials/navbarInstructor.ejs

```diff
@@ -28,23 +42,30 @@
 
         <%# Instance admin %> 
         <li class="navbar-text mx-2 no-select">/</li>
-        <li class="nav-item btn-group">
+        <li class="nav-item btn-group" id="navbar-course-instance-switcher">
             <a class="nav-link <% if (navPage == "instance_admin" && !(typeof navSubPage !== 'undefined' && (navSubPage == "assessments" || navSubPage == "gradebook"))) { %>active<% } %>" href="<%= plainUrlPrefix %>/course_instance/<%= course_instance.id %>/instructor/instance_admin" role="button">
                 <%= course_instance.short_name %>
             </a>
-            <% if (authz_data.course_instances && authz_data.course_instances.length > 0) { %>
-                <a class="nav-link dropdown-toggle dropdown-toggle-split" href="#" id="navbarDropdownMenuInstanceAdminLink" role="button" data-toggle="dropdown" aria-label="Change course instance" aria-haspopup="true" aria-expanded="false">
-                </a>
-                <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuInstanceAdminLink" id="navbarDropdownMenuInstanceAdmin">
-                    <% authz_data.course_instances.forEach(function(ci) { %>
-                        <% if ((navPage == "instance_admin") || (navPage == "course_admin")) { %>
-                            <a class="dropdown-item <% if (course_instance.id == ci.id) { %>active<% } %>" href="<%= plainUrlPrefix %>/course_instance/<%= ci.id %>/instructor/<%= navPage %><% if (typeof navSubPage !== 'undefined' && navSubPage != "file_edit") { %>/<%= navSubPage %><% } %>"><%= ci.short_name %></a>
-                        <% } else { %>
-                            <a class="dropdown-item <% if (course_instance.id == ci.id) { %>active<% } %>" href="<%= plainUrlPrefix %>/course_instance/<%= ci.id %>/instructor/instance_admin"><%= ci.short_name %></a>
-                        <% } %>
```

Comment:
As with my other comment, I opted not to retain this complex behavior in favor of linking unconditionally to the main course instance admin page. Let me know if you want that to be changed.